.github/actions/deploy-action.yaml

```diff
@@ -0,0 +1,30 @@
+name: 'Deploy to Kalm'
```

Comment:
Should we create a actions repo like https://github.com/actions?

Reply:
add a repo for action: https://github.com/kalmhq/kalm-deploy-action

https://github.com/marketplace/actions/kalm-deploy-action